### PR TITLE
[WIP] Adds Workbox Migrate from v5 to v6 guide

### PR DESCRIPTION
**Pull request by housseindjirdeh**
What's changed, or what was fixed?
- Adds a "Migrate from Workbox v5 to v6" guide

**Target Live Date:** TBD

- [ ] This has been reviewed and approved by (NAME)
- [X] I have run `npm test` locally and all tests pass.
- [ ] I have added the appropriate `type-something` label.
- [X] I've staged the site and manually verified that my content displays correctly.

**CC:** @jeffposnick @philipwalton @petele

### CODE REVIEW

File: src/content/en/tools/workbox/guides/migrations/migrate-from-v5.md [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+- Creating a "handler" instance, that can manage state for each individual request a strategy is
+  handling.
+
+#### New "handler" class
+
+We previously had internal modules call `fetchWrapper` and `cacheWrapper`, which (as their name
+implies) wrap the various fetch and cache APIs with hooks into their lifecycle. This is the
+mechanism that currently allows plugins to work, but it's not exposed to developers.
+
+The new "handler" class, `StrategyHandler`, will expose these methods so custom strategies can call
```
Comment (housseindjirdeh) [inline src/content/en/tools/workbox/guides/migrations/migrate-from-v5.md:80]: @jeffposnick release docs mention naming of `StrategyHandler` is in a proposal - so let me know if the name happens to change

**Comment by leyoj5** [comment]
I understand 😄

**Comment by leyoj5** [comment]
I don't know web development a lot 😰

**Comment by leyoj5** [comment]
Apologize me

**Comment by petele** [comment]
@jeffposnick - can you take a look and LGTM/merge please?

**Comment by jeffposnick** [comment]
I'm going to go ahead and merge this, as we've reached the RC stage for v6, and this info could be helpful for early adopters.

We might tweak it a bit once we hit v6.0.0.

**Review by jeffposnick — APPROVED** [review]

**Pull request merged**